File: docs/get-stocks/request.js
import { writeFile } from "fs/promises";

const AUTHCODE = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
const OUTFILE = "get-stocks.xml";

const response = await fetch(`<url-to-rustopus>/get-stocks?authcode=${AUTHCODE}`);
const text = await response.text();

await writeFile(OUTFILE, text);

File: docs/get-barcodes/request.cs
// Get-Barcodes RustOpus example
using System.Net.Http;

var authcode = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
var outFile = "get-barcodes.xml";

using var client = new HttpClient();
var response = await client.GetStringAsync($"https://<url-to-rustopus>/get-barcodes?authcode={authcode}");
await File.WriteAllTextAsync(outFile, response);
